.github/workflows/run_iree_dispatch_profiler.yml

```diff
@@ -0,0 +1,72 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# Workflow for running IREE Dispatch Profiler
+# (https://github.com/openxla/iree/tree/main/experimental/dispatch_profiler).
+#
+# IREE Dispatch Profiler validates functional correctness and performs profiling
+# on matmul, conv, and other ops on IREE CUDA and other back-ends.
+#
+# It uploads results to this a publicly accessible GCS bucket:
+# https://storage.googleapis.com/dispatch-profiler-artifacts
+
+name: dispatch_profiler
+
+on:
+  schedule:
+    - cron: "0 13 * * *"
+  workflow_dispatch:
+  pull_request:
+
+jobs:
+  setup:
+    uses: ./.github/workflows/setup.yml
+
+  build_all:
+    needs: setup
+    if: fromJson(needs.setup.outputs.should-run)
+    uses: ./.github/workflows/build_all.yml
+    with:
+      runner-group: ${{ needs.setup.outputs.runner-group }}
+      runner-env: ${{ needs.setup.outputs.runner-env }}
+      write-caches: ${{ needs.setup.outputs.write-caches }}
```

Comment:
@pzread recommended I use ci.yml/setup. @pzread any thoughts on writing to caches?

Reply:
IMO it should be safe to write caches, as this is running on the merged code (although it might actually do nothing because the commit has been built on the main branch).

Not sure if `setup.yml` is intended to be reused outside `ci.yml`, but it's convenient to get the correct `runner-group` and `runner-env` from it.

Reply:
Yeah setup.yml was pulled out into a callable workflow so that it could be used elsewhere. I'm not sure I would call it "supported", but :shrug: